src/rust/aggregator/src/pipelines/user_history.rs

```diff
@@ -201,9 +202,10 @@ async fn aggregate_fill_for_maker_and_taker<'a>(
     market_id: &BigDecimal,
     time: &DateTime<Utc>,
     price: &BigDecimal,
+    fees: &BigDecimal,
 ) -> PipelineAggregationResult {
-    aggregate_fill(tx, size, maker_order_id, market_id, time, price).await?;
-    aggregate_fill(tx, size, taker_order_id, market_id, time, price).await?;
+    aggregate_fill(tx, size, maker_order_id, market_id, time, price, fees).await?;
```

Comment:
How about passing `0` rather than `fees` for maker side, since makers dont pay fees?

Otherwise, the order history for the maker order will report fees paid as nonzero

Reply:
Right, thanks, updating